checkstyle/import-control.xml

```diff
@@ -63,6 +63,8 @@
       <allow pkg="org.apache.kafka.common.config" />
       <allow pkg="org.apache.kafka.common.metrics" />
       <allow pkg="org.apache.kafka.common.security" />
+      <!-- for testing -->
+      <allow pkg="org.apache.kafka.common.requests" />
```

Comment:
This is not ideal. I still have to check if/how we can get around it, but thought I'd mention it in the meantime in case you have ideas.